docs/guidance/using-goosehints.md

```diff
@@ -0,0 +1,45 @@
+# Using `.goosehints` in Goose
+
+`.goosehints` are text files used within the Goose environment to provide additional context about your project and improve the communication between the developer and Goose. Ensuring it understands your requirements better and can execute tasks more effectively.
+
+>[!TIP]
+> **Developer toolkit required**
+>
+> To make use of the hints file, you need to have the `developer` toolkit enabled.
+
+This guide will walk you through using your `.goosehints` file to streamline your workflow with custom instructions and context.
+
+## Creating your `.goosehints` file
+You can place a `.goosehints` file in your current working directory or globally at `~/.config/goose/.goosehints`. This file can include any repeated instructions or contextual details relevant to your projects.x
```

Comment:
Looks like there's an x at the end of this sentence that may be unnecessary.

Reply:
ooh great catch thanks. A typo.